### PR TITLE
1.1.1 - LATÜRNICH

### PR DESCRIPTION
**Pull request by subwaystation**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/pangenome/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/pangenome _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

This is a small PR which just fixes some bugs.

### CODE REVIEW

File: .github/workflows/download_pipeline.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Install Nextflow
-        uses: nf-core/setup-nextflow@b9f764e8ba5c76b712ace14ecbfcef0e40ae2dd8 # v1
+        uses: nf-core/setup-nextflow@v1
```
Comment (mashehu) [inline .github/workflows/download_pipeline.yml:31]:
```suggestion
        uses: nf-core/setup-nextflow@v2
```
not sure if the linting then complains about it... only one way to find out 🙂

File: .github/workflows/linting.yml [hunk 2]
```diff
@@ -35,7 +35,7 @@ jobs:
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
 
       - name: Install Nextflow
-        uses: nf-core/setup-nextflow@b9f764e8ba5c76b712ace14ecbfcef0e40ae2dd8 # v1
+        uses: nf-core/setup-nextflow@v1
```
Comment (mashehu) [inline .github/workflows/linting.yml:38]:
```suggestion
        uses: nf-core/setup-nextflow@v2
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,15 @@
...
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.1.1 - LATÜRNICH
+
+This release fixes some important bugs:
+
+- Per default, we set the number of mappings in `wfmash` to `1`. Previously, this was set to the given number of haplotypes.
+- To complement the issue above, there is a new parameter `wfmash_n_mappings` with default `1`.
+- `bcftools` in the `VG_DECONSTRUCT` module was updated to the most recent version `1.19` to prevent errors like `corrupted size vs. prev_size`.
+- Fixed some problems with the delimiter in `VG_DECONSTRUCT` so that the parameters given by `--vcf-spec` are now applied correctly.
```
Comment (mashehu) [inline CHANGELOG.md:12]:
```suggestion
- Fixed some problems with the delimiter in `VG_DECONSTRUCT` so that the parameters given by `--vcf-spec` are now applied correctly.
- Updated to nf-core template version 2.13.1
```

Comment (mashehu) [reply CHANGELOG.md:12]: or something like that

File: README.md [hunk 4]
```diff
@@ -7,15 +7,17 @@
...
-[![Launch on Nextflow Tower](https://img.shields.io/badge/Launch%20%F0%9F%9A%80-Nextflow%20Tower-%234256e7)](https://tower.nf/launch?pipeline=https://github.com/nf-core/pangenome)
+[![Launch on Seqera Platform](https://img.shields.io/badge/Launch%20%F0%9F%9A%80-Seqera%20Platform-%234256e7)](https://tower.nf/launch?pipeline=https://github.com/nf-core/pangenome)
 
 [![Get help on Slack](http://img.shields.io/badge/slack-nf--core%20%23pangenome-4A154B?labelColor=000000&logo=slack)](https://nfcore.slack.com/channels/pangenome)[![Follow on Twitter](http://img.shields.io/badge/twitter-%40nf__core-1DA1F2?labelColor=000000&logo=twitter)](https://twitter.com/nf_core)[![Follow on Mastodon](https://img.shields.io/badge/mastodon-nf__core-6364ff?labelColor=FFFFFF&logo=mastodon)](https://mstdn.science/@nf_core)[![Watch on YouTube](http://img.shields.io/badge/youtube-nf--core-FF0000?labelColor=000000&logo=youtube)](https://www.youtube.com/c/nf-core)
 
+> WARNING: Version 1.1.0 does contain some known bugs. It is recommended to directly use version 1.1.1. For details please take a look at [CHANGELOG.md](CHANGELOG.md).
+
```
Comment (mashehu) [inline README.md:20]:
```suggestion
> [!WARNING]
> Version 1.1.0 contains some known bugs. It is recommended to directly use version 1.1.1. For details please take a look at [CHANGELOG.md](CHANGELOG.md).

```
but I actually don't think this is necessary, the general recommendation is to always use the latest version of a pipeline and it's easy to forget to remove these section with the next release.

File: nextflow_schema.json [hunk 5]
```diff
@@ -255,8 +260,8 @@
             "properties": {
                 "vcf_spec": {
                     "type": "string",
-                    "description": "Specify a set of VCFs to produce with --vcf_spec \"REF:DELIM[:LEN][,REF:DELIM:[LEN]]*\".",
-                    "help_text": "The paths matching ^REF are used as a reference, while the sample haplotypes are derived from path names, e.g. when DELIM=# and with '-V chm13:#', a path named HG002#1#ctg would be assigned to sample HG002 phase 1. If LEN is specified and greater than 0, the VCFs are decomposed, filtering sites whose max allele length is greater than LEN."
+                    "description": "Specify a set of VCFs to produce with --vcf_spec \"REF[:LEN][,REF[:LEN]]*\".",
```
Comment (mashehu) [inline nextflow_schema.json:263]:
```suggestion
                    "description": "Specify a set of VCFs to produce with `--vcf_spec \"REF[:LEN][,REF[:LEN]]*\"`.",
```

File: nextflow_schema.json [hunk 6]
```diff
@@ -255,8 +260,8 @@
...
+                    "help_text": "The paths matching ^REF are used as a reference, while the sample haplotypes are derived from path names, e.g. when DELIM=# and with '-V chm13', a path named HG002#1#ctg would be assigned to sample HG002 phase 1. If LEN is specified and greater than 0, the VCFs are decomposed, filtering sites whose max allele length is greater than LEN."
```
Comment (mashehu) [inline nextflow_schema.json:264]:
```suggestion
                    "help_text": "The paths matching `^REF` are used as a reference, while the sample haplotypes are derived from path names, e.g. when `DELIM=#` and with `-V chm13`, a path named HG002#1#ctg would be assigned to sample HG002 phase 1. If `LEN` is specified and greater than 0, the VCFs are decomposed, filtering sites whose max allele length is greater than `LEN`."
```

**Review by mashehu — CHANGES_REQUESTED** [review]

File: .github/workflows/download_pipeline.yml (hunk 1 above)
Comment (subwaystation) [reply .github/workflows/download_pipeline.yml:31]: sure.

File: README.md (hunk 4 above)
Comment (subwaystation) [reply README.md:20]: aah, so let me remove this

File: .github/workflows/download_pipeline.yml (hunk 1 above)
Comment (subwaystation) [reply .github/workflows/download_pipeline.yml:31]: this one didn't work, but the other one.

Comment (subwaystation) [reply .github/workflows/download_pipeline.yml:31]: whoopsy, let me try again

**Comment by subwaystation** [comment]
@mashehu The linter was only unhappy with `.github/workflows/linting.yml`. I think I added all your suggestions, thanks!

**Review by mashehu — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -18,7 +18,7 @@ Learn more about contributing: [CONTRIBUTING.md](https://github.com/nf-core/pang
 - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/pangenome/tree/master/.github/CONTRIBUTING.md)
 - [ ] If necessary, also make a PR on the nf-core/pangenome _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
 - [ ] Make sure your code lints (`nf-core lint`).
-- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
+- [ ] Ensure the test suite passes (`nf-test test main.nf.test -profile test,docker`).
```
Comment (heuermh) [inline .github/PULL_REQUEST_TEMPLATE.md:21]: I assume this is a template change, but is pangenome ready for `nf-test` yet?

**Review by heuermh — APPROVED** [review]
Tests ok for me on Docker!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 7 above)
Comment (subwaystation) [reply .github/PULL_REQUEST_TEMPLATE.md:21]: This is a template change. I did not update tests so they can be run with nf-test. Let's keep this open for next week's hackathon. I will open an issue with details and hopefully someone will take care of this ;)

**Comment by subwaystation** [comment]
Thanks for the approval @heuermh !
